packages/cli/src/WorkflowHelpers.ts

```diff
@@ -250,11 +239,11 @@ export async function saveStaticData(workflow: Workflow): Promise<void> {
 /**
  * Saves the given static data on workflow
  *
- * @param {(string | number)} workflowId The id of the workflow to save data on
+ * @param {(string)} workflowId The id of the workflow to save data on
```

Comment:
I thought we had gotten rid of redundant JSDoc.

Reply:
I thought so too, but there are still way too many of them to clean up in this PR.